x-pack/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -65,6 +66,12 @@ function validateConfig(
 ): string | void {
   const config = configObject;
 
+  const emails = [config.from];
+  const invalidEmailsMessage = configurationUtilities.validateEmailAddresses(emails);
+  if (!!invalidEmailsMessage) {
```

Comment:
Would it be useful to identify whether the invalid email address is in the `from` configuration or the `to/cc/bcc` configuration? I think sometimes the users that configure the connectors differ from the users that configure rules with actions so this could give some clarity into which part of the connector needs to be updated.

Reply:
I was a little worried about being super-precise on which emails of which fields were responsible for errors, in terms of making the messages even more incomprehensible than they will be.  But worth another peek to see if it's easy.  Due to the time constraint (would like to get this merged for 7.17.4 tomorrow), thinking this is also something we could defer ...

Reply:
I was able to make a simple change to have different messages for the from and for the to/cc/bcc errors.  The to/cc/bcc are still lumped together, in the output message.  We could improve this later, but was starting to get messy - will require a bit more work to separate them out by to/cc/bcc, and not sure it's worth it.

Reply:
I think it's fine to lump the to/cc/bcc together! I felt we should differentiate the from because it is set in an entirely separate place from the other fields.